x-pack/plugins/canvas/types/embeddables.ts

```diff
@@ -8,9 +8,12 @@
 import type { TimeRange } from '@kbn/es-query';
 import { Filter } from '@kbn/es-query';
 import { EmbeddableInput as Input } from '@kbn/embeddable-plugin/common';
+import { CanAddNewPanel, PublishesViewMode } from '@kbn/presentation-publishing';
 
 export type EmbeddableInput = Input & {
   timeRange?: TimeRange;
   filters?: Filter[];
   savedObjectId?: string;
 };
+
+export type CanvasContainerApi = PublishesViewMode & CanAddNewPanel;
```

Comment:
I think it's pretty powerful that we can add a `parent API` to Embeddables this easily. In some other page, we could even use this to provide context / filters / etc.